### PR TITLE
Added new resource: tags

### PR DESCRIPTION
**Pull request by bjoernmayer**
Addes the new availabe tags resource

### CODE REVIEW

File: example/web/tags.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ */
+
+$config = require_once __DIR__ . '/../config.php';
+$tas    = new \Trivago\Tas\Tas($config);
+
+$tagGroups = $tas->getTags(new \Trivago\Tas\Request\TagsRequest());
+
+echo '<pre>';
+var_dump($tagGroups);
+echo '</pre>';
```
Comment (markuspoerschke) [inline example/web/tags.php]: No newline at file ending ;)

File: src/Response/Tags/TagGroups.php [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+ * limitations under the License.
+ */
+
+namespace Trivago\Tas\Response\Tags;
+
+use Trivago\Tas\Response\Response;
+
+class TagGroups implements \Iterator, \Countable
+{
+
```
Comment (Exeu) [inline src/Response/Tags/TagGroups.php]: drop empty l ine

File: src/Response/Tags/TagGroups.php [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+     * @var TagGroup[]
+     */
+    private $tagGroups = [];
+
+    private function __construct()
+    {
+        // intentionally left empty. use named constructor to create new instances.
+    }
+
+    public static function fromResponse(Response $response)
```
Comment (Exeu) [inline src/Response/Tags/TagGroups.php]: Please at a docblock to this function. 
Maybe briefly describe how the return value looks like in the doc block.

File: docs/reference/trivago/tas/response/tags/tag-group.rst [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+getType()
+=========
+
+.. code-block:: php
+
+    public function string getType()
+
+Returns the type of the tag group. This can be one of this values:
+
+- and
```
Comment (Exeu) [inline docs/reference/trivago/tas/response/tags/tag-group.rst]: Briefly describe what "and" means (also if it seems to be clear).

File: docs/reference/trivago/tas/response/tags/tag-group.rst [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+- or
```
Comment (Exeu) [inline docs/reference/trivago/tas/response/tags/tag-group.rst]: Briefly describe what "or" means (also if it seems to be clear).

**Pull request merged**